code/modules/clothing/under/striketeam.dm

```diff
@@ -0,0 +1,143 @@
+var/list/deathsquad_uniforms = list()
+
+var/list/holomap_cache = list()
+
+/obj/item/clothing/under/deathsquad
+	name = "deathsquad holosuit"
+	desc = "A state-of-the-art suit featuring an holographic map of the area, to help the squad coordinate their efforts."
+	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/altsuits.dmi', "right_hand" = 'icons/mob/in-hand/right/altsuits.dmi')
+	icon_state = "deathsquad"
+	item_state = "deathsquad"
+	_color = "deathsquad"
+	flags = FPRINT  | ONESIZEFITSALL
+	action_button_name = "Toggle Holomap"
+
+/obj/item/clothing/under/deathsquad/New()
+	..()
+	deathsquad_uniforms += src
+
+/obj/item/clothing/under/deathsquad/Destroy()
+	deathsquad_uniforms -= src
+
+	for(var/cacheIcon in holomap_cache)
+		if(findtext(cacheIcon, "\ref[src]"))
+			var/image/I = holomap_cache[cacheIcon]
+			animate(I)
+			qdel(I)
```

Comment:
Ah you shouldn't actually 'qdel' these since qdel is undefined for images, just making them garbage collect is good enough